### DISCUSSION TITLE
More powerful CRUDBase

### DISCUSSION THREAD

**Discussion by wu-clan** [fastapi-practices/fastapi_best_architecture#267] [General]
Changing existing behaviors to make them more generic and robust.

**Comment by jundaychan** [comment]
纯 sqlalchemy 心智负担有点重，我看你们在讨论 sqlmodel，我试用了下感觉确实比sqlalchemy 学习曲线好一点。

**Reply by wu-clan** [reply]
我同意你的看法，并且，吐槽过 sqla 的文档，但它是主流，并且更原生，如果你在 sqla 官方仓库中问过问题，我相信，官方回复速度没有任何一个其他 ORM 能与之抗衡，这是我坚持的理由

**Reply by jundaychan** [reply]
ok 我也同意

**Comment by wu-clan** [comment]
https://github.com/fastapi-practices/sqlalchemy-crud-plus
